x-pack/plugin/eql/qa/correctness/README.md

```diff
@@ -85,3 +85,10 @@ Once the data is loaded, a specific query can be run against the running ES node
 ```
 
 **Set the `eql_test_credentials_file` environmental variable correctly in the shell before running the command above,**
+
+#### Preserve data across node restarts
+If you'd like to preserve the restored index and avoid the network download and delay of restoring them on every run of the node,
+you can set the `eql.test.preserve.data` system property, e.g.:
+```shell script
+./gradlew :x-pack:plugin:eql:qa:correctness:javaRestTest -Deql.test.preserve.data=true
```

Comment:
Why do we need the special eql variable and not use `--preserve-data` ?

Reply:
I also tried it with `-Deql.test.preserve.data=true` and doesn't work, where with `--preserve-data` works.
I think the issue is:
```
if (Boolean.parseBoolean(System.getProperty('eql.test.preserve.data', 'false'))) {
->  preserveDataDir = true
}
```
this doesn't the variable correctly, but again I think we can just use --preserve-date and not add an custom variable for this project.

Reply:
Using `--preserve-data` is only when using the run task. So that's when running the `runEqlCorrectnessNode`, if running `javaRestTest` you need to use the system property. The difference is because they are two different implementations.

Reply:
Ah, I see, I mixed it, let me just double check, thx!

Reply:
I think it's better to move this paragraph before `### Run an ES node manually and run the tests against it`
and add a new paragraph here at the end to mention the --preserve-data for the `./gradlew :x-pack:plugin:eql:qa:correctness:runEqlCorrectnessNode`:

````
#### Preserve data across node restarts
If you'd like to preserve the restored index and avoid the network download and delay of restoring them on every run of the node, you can start the node with:

```shell script
./gradlew :x-pack:plugin:eql:qa:correctness:runEqlCorrectnessNode --debug-jvm --preserve-data
```
````
I could do it myself in a followup PR though.